src/FakeItEasy/ArgumentConstraintManagerExtensions.cs

```diff
@@ -157,6 +157,19 @@ public static string IsNullOrEmpty(this IArgumentConstraintManager<string> manag
         }
 
         /// <summary>
+        /// The tested argument collection should contain the same elements as the
+        /// as the specified collection.

```

Comment:
"as the as the". As I comment, I see that it appears that way above, as well.
While I'm complaining, I wonder if we shouldn't mention that we expect the elements to be in the same order.

Reply:
Ah, yes. I just copy-pasted the doc comment and didn't notice the problem. I'll fix it.